### PR TITLE
new book - Tales of a Tired Janitor

### PR DESCRIPTION
**Pull request by Boaz1111**
## About the PR

I added a new book, as requested by someone in the discord. This book is about the life of a janitor working aboard a (presumably NT) space station "14". They describe the things they have to deal with, the disrespect from others, the lack of good equipment.
## Why / Balance

More variety in books, especially books indirectly talking about life on a space station, and the jobs on said station.
## Media

- [X] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

## Breaking changes

**Changelog**

no cl

### CODE REVIEW

File: Resources/Locale/en-US/paper/book-authorbooks.ftl [hunk 1]
```diff
@@ -253,3 +253,12 @@ book-text-inspiration = As a freelance writer, Sarah was always in search of ins
       Without a moment's hesitation, Sarah took out her pen and notebook and began to write. She wrote about the beauty of the scene, the peacefulness she felt, and the surreal quality of the moment. She wanted to capture this feeling and share it with others through her words.
       As she finished writing, Sarah heard the snap of a twig and looked up to see a man approaching her with a camera. He introduced himself as a nature photographer and asked if she had seen the waterfall. They struck up a conversation, and soon they were both laughing and sharing stories of their adventures.
       As the sun began to set, the two of them packed up their gear and said their goodbyes. Sarah felt grateful for this unexpected encounter and the inspiration it brought her. She knew that this moment would stay with her forever, and she felt a sense of serenity knowing that there was still so much beauty in the world waiting to be discovered.
+
+book-text-janitor = [bold][color=black][head=3]The Tales of a Tired Janitor[/head][/bold]
+      Written by Caleb-Maxath
```
Comment (GoodWheatley) [inline Resources/Locale/en-US/paper/book-authorbooks.ftl:258]: Don’t reference your character.

**Review by GoodWheatley — CHANGES_REQUESTED** [review]

File: Resources/Locale/en-US/paper/book-authorbooks.ftl (hunk 1 above)
Comment (Boaz1111) [reply Resources/Locale/en-US/paper/book-authorbooks.ftl:258]: it's not my book, someone asked me to add it, will remove

**Review by keronshb — DISMISSED** [review]
Please be more descriptive with your PR.

**Comment by Boaz1111** [comment]
my apologies, I didn't have much time to write the description previously.

**Comment by keronshb** [comment]
> my apologies, I didn't have much time to write the description previously.

The description is still too vague, including the PR title. Please title the PR what the new book is and in the PR explain it.

**Pull request merged**